bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,374 @@
+    async def company(self, ctx: Context, amount: int = 5) -> None:
+        """
+        Get random Game Companies companies from IGDB API.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if not 1 <= amount <= 25:
+            await ctx.send("Your provided amount is out of range. Our minimum is 1 and maximum 25.")
+            return
+
+        companies = await self.get_companies_list(amount, random.randint(0, 150))
+        pages = [await self.create_company_page(co) for co in companies]
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title="Random Game Companies"))
+
+    async def get_games_list(self,
+                             amount: int,
+                             genre: Optional[str] = None,
+                             sort: Optional[str] = None,
+                             additional_body: str = "",
+                             offset: int = 0
+                             ) -> List[Dict[str, Any]]:
+        """
+        Get list of games from IGDB API by parameters that is provided.
+
+        Amount param show how much movies this get, genre is genre ID and at least one genre in game must this when
+        provided. Sort is sorting by specific field and direction, ex. total_rating desc/asc (total_rating is field,
+        desc/asc is direction). Additional_body is field where you can pass extra search parameters. Offset show start
+        position in API.
+        """
+        # Create body of IGDB API request, define fields, sorting, offset, limit and genre
+        params = {
+            "sort": f"sort {sort};" if sort else "",
+            "limit": f"limit {amount};",
+            "offset": f"offset {offset};" if offset else "",
+            "genre": f"where genres = ({genre});" if genre else "",
+            "additional": additional_body
+        }
+        body = GAMES_LIST_BODY.substitute(params)
+
+        # Do request to IGDB API, create headers, URL, define body, return result
+        async with self.http_session.get(url=f"{BASE_URL}/games", data=body, headers=HEADERS) as resp:
+            return await resp.json()
+
+    async def create_page(self, data: Dict[str, Any]) -> Tuple[str, str]:
+        """Create content of Game Page."""
+        # Create cover image URL from template
+        url = COVER_URL.substitute({"image_id": data["cover"]["image_id"] if "cover" in data else ""})
+
+        # Get release date separately with checking
+        release_date = dt.utcfromtimestamp(data["first_release_date"]).date() if "first_release_date" in data else "?"
+
+        # Create Age Ratings value
+        rating = ", ".join(f"{AgeRatingCategories(age['category']).name} {AgeRatings(age['rating']).name}"
+                           for age in data["age_ratings"]) if "age_ratings" in data else "?"
+
+        companies = ", ".join(comp["company"]["name"] for comp in data["involved_companies"]) \
+            if "involved_companies" in data else "?"
+
```

Comment:
I do not like the way the multi-line has been handled. they are inconsistent and overly messy. 
I do not have a suggestion atm, but we can have a discussion about it at your leisure. The contribution guide might have a suggestion about what to do.